tests/lsp/test_selection_range.py

```diff
@@ -73,7 +73,7 @@ def test_selection_range_return_list(client_server):
     response = client.lsp.send_request(
         SELECTION_RANGE,
         SelectionRangeParams(
-            query="query",
+            # query="query",
```

Comment:
Perhaps I just missed it, but I couldn't find a type definition in `lsprotocol` that had this field, nor could I find it in the [spec](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#selectionRangeParams).

Does anyone know why this is here?

Reply:
I guess this is something that might be answered by brave implementers of the release candidate?